bindings/protocols/profinet/index.html

```diff
@@ -0,0 +1,1040 @@
+                communication,. The arrangement of bytes and words may vary between systems, and these properties provide a declarative way to capture such configurations. 
+                For instance, in Big-Endian byte order (e.g., AABBCC), the most significant byte comes first, and setting <code>profv:mostSignificantByte</code> to true ensures the correct
+                interpretation of the data as AABBCC. Conversely, in Little-Endian byte order (e.g., CCBBAA), the least significant
+                byte comes first, therefore <code>profv:mostSignificantByte</code> has to be set to false.
+                On the other hand, the <code>profv:mostSignificantWord</code> property extends this functionality to message payloads
+                composed by more than two registers. For example, given a message payload composed by four registers (e.g., AABBCCDD),
+                with <code>profv:mostSignificantWord</code> set to true let the client to correctly interpretate the data as AABBCCDD, 
+                while setting the property to false will result in the interpretation of the data as CCDDAABB. 
+                Though, almost all implemtnetations provided by vendors are in Big-Endian but PROFINET specification does not specifically adopt Big-Endianness.
+                So, some off-the-shelf implementation or user specific implementaiton might adopt a different endianness.
+                Therefore, when creating a Thing Description (TD), users may need to account for such deviations by configuring the
+                <code>profv:mostSignificantWord</code> and <code>profv:mostSignificantByte</code> properties accordingly.
+            </p>
+
+        </section>
+        <section id="payloaddatatype">
+            <h3>Payload DataType</h3>
+            <p>
+                The <code>PayloadDataType</code> class within the PROFINETE Binding Template serves as value for the <code>profv:type</code> property to specify
+                the expected data types of payload content in PROFINET messages. It offers a set of terms taken from PROFINET specification (chapter 5 of IEC 61158-5-10)[ref] to
+                cover the most common data types used in PROFINET messages. Note that the <code>PayloadDataType</code> 
+                entity is designed for describing established conventions in the PROFINET ecosystem, further development might remove this 
+                functionality or add new terms.
+                Currently, the <code>PayloadDataType</code> class contains the following data types:
+            </p>
+            <table class="def">
+                <thead>
+                    <tr>
+                        <th>Value</th>
+                        <th>Byte Length</th>
+                        <th>Code</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <!-- Generate forms terms from the ontology-->
+                    
+        <tr>
```

Comment:
In this section, don't we need a mapping of how these data are mapped to the JSON representation on the InteractionAffordances?